packages/consts/src/knight-hacks.ts

```diff
@@ -36,20 +37,34 @@ export const EVENT_POINTS: Record<EventTag, number> = {
   Sponsorship: 40,
   "Tech Exploration": 25,
   "Class Support": 25,
+  Workshop: 25,
   OPS: 20,
   Hackathon: 1,
 } as const;
 
 export const KNIGHTHACKS_MEMBERSHIP_PRICE = 2500;
 
+// PROD: 1319413082258411652
+// DEV: 1321955700540309645
 export const DISCORD_ADMIN_ROLE_ID = "1319413082258411652";
 
+// PROD: 486628710443778071
+// DEV: 1151877367434850364
 export const KNIGHTHACKS_GUILD_ID = "486628710443778071";
 
 export const DISCORD_EVENT_TYPE = 3;
 
 export const DISCORD_EVENT_PRIVACY_LEVEL = 2;
 
+// PROD: c_0b9df2b0062a5d711fc16060ff3286ef404b174bfafc4cbdd4e3009e91536e94@group.calendar.google.com
+// DEV: c_178118a9a25d9f278014b123b79b7e9caf9b29ac94bba3934237db00979e0f75@group.calendar.google.com
+export const GOOGLE_CALENDAR_ID =
+  "c_0b9df2b0062a5d711fc16060ff3286ef404b174bfafc4cbdd4e3009e91536e94@group.calendar.google.com";
```

Comment:
If these differentiate between prod and dev we might wanna make them env variables